swahili-syntax/syntaxes/swahili.tmLanguage.json

```
@@ -0,0 +1,113 @@
+{
+	"$schema": "https://raw.githubusercontent.com/martinring/tmlanguage/master/tmlanguage.json",
+	"name": "swahili",
+	"patterns": [
+		{
+			"include": "#keywords"
+		},
+		{
+			"include": "#strings"
+		},
+		{
+			"include": "#numerical values"
+		},
+		{
+			"include": "#Functions"
+		},
+		{
+			"include": "#single line comments"
+		},
+		{
+			"include": "#Multi-line comments"
+		},
+		{
+			"include": "#Flow Control"
+		},
+		{
+			"include": "#Function Names"
+		},
+		{
+			"include": "#Identifier"
+		},
+		{
+			"include": "#Function Call"
+		}
+	],
+	"repository": {
+		"keywords": {
+			"patterns": [{
+				"name": "keyword.control.swahili",
+				"match": "\\b(wacha|shughuli|kweli|uwongo)\\b"
```

Add `tupu` as a keyword
```suggestion
				"match": "\\b(wacha|shughuli|kweli|uwongo|tupu)\\b"
```

---

Do I also remove it  from functions?

---

I'm shooting to get rid of that block entirely because there isn't any need to highlight built in function calls differently from regular ones